lidar_camera_calibration_py/tests/test_config_utils.py

```
@@ -0,0 +1,47 @@
+    Test parse_config_file raises FileNotFoundError for missing file.
+    """
+    with pytest.raises(FileNotFoundError):
+        config_utils.parse_config_file('nonexistent_config.txt')
+
+def test_parse_marker_coordinates_malformed(tmp_path) -> None:
+    """
+    Test parse_marker_coordinates handles malformed file gracefully.
+    """
+    malformed = tmp_path / "malformed.txt"
+    malformed.write_text("not a number\nfoo\nbar\n")
+    with pytest.raises(Exception):
```

Ruff (0.14.0)

Do not assert blind exception: `Exception`

(B017)